source/releasenotes/2025-02-27-papc-2.1.1.md

```diff
@@ -0,0 +1,14 @@
+---
+title: WordPress Pantheon Advanced Page Cache 2.1.1 update now available
+published_date: "2025-02-27"
+categories: [wordpress, plugins]
+---
+
+The WordPress Pantheon Advanced Page Cache [v2.1.1](https://github.com/pantheon-systems/pantheon-advanced-page-cache/releases) update is now available. This latest update resolves a bug that caused some sites to cache 404 pages, leaving scheduled posts unavailable after publishing.
+
+## What's new?
+
+* A post's path is explicitly purged when it is published for the first time, in addition to related surrogate keys
+* Paths purged can be further customized with the `pantheon_clear_post_path` [filter](https://github.com/pantheon-systems/pantheon-advanced-page-cache?tab=readme-ov-file#additional-purging-by-path).
```

Comment:
"paths purged" sounds clumsy. flipping them makes more sense and doesn't lose the meaning.

```suggestion
* Purged paths can be further customized with the `pantheon_clear_post_path` [filter](https://github.com/pantheon-systems/pantheon-advanced-page-cache?tab=readme-ov-file#additional-purging-by-path).
```